config/docker/thinking_sphinx.yml

```diff
@@ -1,44 +1,43 @@
-common: &sphinx
-  configuration_file: <%= ENV.fetch('THINKING_SPHINX_CONFIGURATION_FILE') { Rails.root.join("config/#{Rails.env}.sphinx.conf")} %>
+production:
+  # ----------------------------------
+  # Thinking Sphinx configuration
+  # see https://gist.github.com/pat/a7d73376dd657b4457092efc9e9c418a
+  # ----------------------------------
+  mysql41: 9306
+  address: <%= ENV['THINKING_SPHINX_ADDRESS'] || '0.0.0.0' %>
```

Comment:
This looks like daemon address. `0.0.0.0` means bind to all IPs of host.

Reply:
I am not sure why this is so, it used to be like that.
The 3scale operator is overriding this anyway, by settings to [0.0.0.0](https://github.com/3scale/3scale-operator/blob/7d484662c79a0363e168e08b214f5a7bbf7fb771/pkg/3scale/amp/component/system.go#L170) for the sphinx pod, and [system-sphinx](https://github.com/3scale/3scale-operator/blob/7d484662c79a0363e168e08b214f5a7bbf7fb771/pkg/3scale/amp/component/system.go#L210) for other system-app pods.

Reply:
it makes sense to be `0.0.0.0`. Because we don't know IP in advance. My point was about your grouping, that perhaps it should go under daemon settings.

Reply:
Aaah, OK.
Re: grouping, I used the following rules:

1. If it appears in https://gist.github.com/pat/a7d73376dd657b4457092efc9e9c418a, it's "thinking sphinx" config (`address` specifically is used not only for the daemon, but also for the client)
2. If it's not "thinking sphinx", then it belongs to the group that the config appears in in the Sphinx documentation: https://sphinxsearch.com/docs/current.html

Reply:
fine with me, apparently it can't be completely unambiguous

So I only request info to go to commit message, other than that LGT

Reply:
M